src/edu/wright/cs/raiderplanner/controller/TaskController.java

```diff
@@ -164,7 +157,31 @@ public void handleChange() {
 		}
 		// =================
 	}
-
+	
+	/**
+	 * Used to test all user entries needed before allowing the ok/submit button to be 
+	 * 	pressed.  
+	 * @return true if unlock is successful, or false if not
+	 */
+	public boolean unlockSubmit() {
+		if (!this.name.getText().trim().isEmpty()
```

Comment:
Are we sure that the text field contents will always be non-null?  Perhaps we should something like `this.name.getText() != null && this.name.getText.trim().isEmpty() ....`